### PR TITLE
 Update `Path`  title for podman binary 

### LINKED ISSUE #3847: "Path" title for podman binary should be "Path to Binary"
### Is your enhancement related to a problem? Please describe

"Path" title for the configuration setting is a bit ambiguous.

### Describe the solution you'd like

Change to "Path to Binary" or something similar.

```
    "configuration": {
      "title": "Podman",
      "properties": {
        "podman.binary.path": {
          "name": "Path to Podman Binary",
          "type": "string",
          "format": "file",
          "default": "",
          "description": "Custom path to Podman binary (Default is blank)"
        },
```

To `podman.binary.pathToBinary`

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by aditipatelpro**
Signed-off-by: aditipatelpro <aditipatelpro@gmail.com>

### What does this PR do?
Updates the  `Path` title name for  podman binary
### What issues does this PR fix or reference?
Closes #3847

### CODE REVIEW

File: extensions/podman/src/podman-cli.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ export function getPodmanCli(): string {
 // Get the Podman binary path from configuration podman.binary.path
 // return string or undefined
 export function getCustomBinaryPath(): string | undefined {
-  return extensionApi.configuration.getConfiguration('podman').get('binary.path');
+  return extensionApi.configuration.getConfiguration('podman').get('binary.pathToBinary');
```
Comment (benoitf) [inline extensions/podman/src/podman-cli.ts:54]: maybe we need in the extension startup, check if we have the old property name and migrate it to the new name as well ?

Comment (aditipatelpro) [reply extensions/podman/src/podman-cli.ts:54]: Hi @benoitf, I have checked references for the old property name, `podman-cli.ts`is the only place I found the reference to `binary.path`.

I believe this feature was introduced in #941. I looked at this  PR and found that `binary.path` was only referenced in `podman-cli-.ts`.

**FYI** - I am fairly new to the codebase and open-source contribution, so please let me know if I have missed anything!

Comment (benoitf) [reply extensions/podman/src/podman-cli.ts:54]: so here, probably we would need to check for an old value and then save it to the new value or check both old and new value

**Comment by benoitf** [comment]
Here is an example where property has been changed /renamed

https://github.com/containers/podman-desktop/pull/3840

**Comment by benoitf** [comment]
Here are the interesting lines https://github.com/containers/podman-desktop/blob/ae55b764575024f56d75e700801063d09ca52cb8/packages/main/src/plugin/autostart-engine.ts#L43-L48

Where we check old value

File: extensions/podman/src/podman-cli.ts (hunk 1 above)
Comment (benoitf) [reply extensions/podman/src/podman-cli.ts:54]: @aditipatelpro thanks for contributing, I have added some pointers to guide you on how to read older configuration value ( let say a user is migrating from an old version of Podman Desktop to a newer version where the property is now updated)

Comment (aditipatelpro) [reply extensions/podman/src/podman-cli.ts:54]: @benoitf, thanks for guiding. Let me look at the pointers you suggested. Thanks again.

**Comment by benoitf** [comment]
closing, stale PR

**Pull request closed without merging**